.github/workflows/git-artifacts.yml

```
@@ -388,7 +448,8 @@ jobs:
         run: |
           & .\git-sdk-${{matrix.arch.bitness}}-build-installers\usr\bin\bash.exe -lc @"
             set -x
-            /usr/src/build-extra/please.sh make_installers_from_mingw_w64_git --version=`$(cat pkg-${{matrix.arch.name}}/ver) -o artifacts --${{matrix.artifact.name}} --pkg=pkg-${{matrix.arch.name}}/mingw-w64-${{matrix.arch.name}}-git-[0-9]*.tar.xz --pkg=pkg-${{matrix.arch.name}}/mingw-w64-${{matrix.arch.name}}-git-doc-html-[0-9]*.tar.xz &&
+            [[ \"${{matrix.arch.arm64}}\" = true ]] && ARM64=\"--include-arm64-artifacts=\"`$PWD/arm64\"\" || ARM64=
```

I think those quotes should not be escaped, right?

---

Oh, forget what I said, this is inside a PowerShell step. But that means that there needs to be extra quoting around the path, like so:
```suggestion
            [[ \"${{matrix.arch.arm64}}\" = true ]] && ARM64=\"--include-arm64-artifacts=\\\"`$PWD/arm64\\\"\" || ARM64=
```

---

Interesting. It worked without the quotes, but with the quotes it breaks the installer build: https://github.com/dennisameling/git/runs/1881018607?check_suite_focus=true

Looks like we need to tweak the `release.sh` script in `build-extra/installer` a bit. Will have a look tomorrow.